frontend/packages/shared/src/hooks/queries/useAppMetadataModelIdsQuery.ts

```diff
@@ -3,10 +3,14 @@ import type { UseQueryResult } from '@tanstack/react-query';
 import { useServicesContext } from 'app-shared/contexts/ServicesContext';
 import { QueryKey } from 'app-shared/types/QueryKey';
 
-export const useAppMetadataModelIdsQuery = (org: string, app: string): UseQueryResult<string[]> => {
+export const useAppMetadataModelIdsQuery = (
+  org: string,
+  app: string,
+  onlyUnReferenced: boolean = false,
```

Comment:
Should this be default `true`? Since we pass `false` when getting `allDataModelIds` and no param when getting `availableDataModelIds`

Reply:
That's great spotted! 🚀